src/scaffoldmaker/utils/cylindermesh.py

```diff
@@ -425,14 +425,16 @@ class Ellipse2D:
     def __init__(self, centre, majorAxis, minorAxis,
                  elementsCountAcrossMajor, elementsCountAcrossMinor, elementsCountAcrossShell,
                  elementsCountAcrossTransition, shellProportion, coreMajorRadius, coreMinorRadius,
-                 ellipseShape=EllipseShape.Ellipse_SHAPE_FULL):
+                 ellipseShape=EllipseShape.Ellipse_SHAPE_FULL,
+                 isCore=False):
         """
         :param centre: Ellipse centre.
```

Comment:
I think isCore parameter can be removed since the new solid core doesn't use Ellipse2D anymore.